src/doc_builder/external.py

```diff
@@ -80,22 +83,25 @@ def get_stable_version(package_name):
     return "main"
 
 
-def get_objects_map(package_name, version="main", language="en"):
+def get_objects_map(package_name, version="main", language="en", repo_owner="huggingface", repo_name=None):
     """
     Downloads the `objects.inv` for a package and post-processes it to get a nice dictionary.
 
     Args:
         package_name (`str`): The name of the external package.
         version (`str`, *optional*, defaults to `"main"`): The version of the package for which documentation is built.
         language (`str`, *optional*, defaults to `"en"`): The langauge of the documentation being built.
+        repo_owner (`str`, *optional*, defaults to `"huggingface"`): The owner of the GitHub repo.
+        repo_name (`str`, *optional*): The name of the GitHub repo. If not provided, will be the same as the package name.
     """
+    repo_name = repo_name or package_name
     # We link to main in `package_name` from the main doc (or PR docs) but to the last stable release otherwise.
     if version in ["main", "master"] or version.startswith("pr_"):
         package_version = "main"
     else:
-        package_version = get_stable_version(package_name)
+        package_version = get_stable_version(package_name, repo_owner, repo_name)
 
-    doc_url = f"{HF_DOC_PREFIX}{package_name}/{package_version}/{language}"
+    doc_url = f"{HF_DOC_PREFIX}{repo_name}/{package_version}/{language}"
```

Comment:
Are we sure that we want the doc to be hosted in the `repo_name` env? I'd expect hf.co/docs/timm, not hf.co/docs/pytorch-pretrained-models

Reply:
good catch!!! Reading back through my logic here, I think this was a mistake